website/docs/Basic_operations/read.md

```diff
@@ -85,10 +85,117 @@ Here is a list of the most commonly used operators.
 
 `$eq`: select records that are equal to a specific value
 
+### Retrieve documents containing a value in an array
+
+Insert the following documents in an `employees` collection using this command:
+
+```js
+db.employees.insertMany([
```

Comment:
I'm not sure if in this example formatting is consistent, e.g. whitespace seem to be missing to me.
I've see we have this issue https://github.com/FerretDB/FerretDB/issues/1389, so hopefully, we can automate formatting.